theurgy.lic

```diff
@@ -290,6 +290,10 @@ class Theurgy
 
   def study_wall
     return if DRStats.circle < 30
+    get_data('spells')
+      .spell_data
+      .select { |_name, properties| properties['cyclic'] }
+    return unless get_data('spells').spell_data.select { |name, _properties| DRSpells.active_spells.keys.include?(name) }.empty?
```

Comment:
It's very similar to how common-arcana does release cyclics, but I'm only trying to check if a cyclic is running here, which doesn't have a method built into common-arcana.  Though we could push the method over there and just call it from here.  I wasn't sure how often we would need to only check if a cyclic was running.